packages/babel-parser/test/fixtures/es2015/destructuring/parenthesized-lhs-array/options.json

```diff
@@ -1,3 +1,3 @@
 {
-  "throws": "You're trying to assign to a parenthesized expression, eg. instead of `([a]) = 0` use `([a] = 0)` (1:1)"
```

Comment:
We lose this specific error message by applying a more general fix but I think it's probably worth it in terms of code cleanliness - specific cases can be added later on.